app.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "kepler",
+  "buildpacks": [{
+    "url": "https://github.com/dulaccc/heroku-buildpack-geodjango.git#1.1"
```

Comment:
Maybe. I've only used app.json to make the automatic PR builds work in pipelines...